tools/operator/README.md

```diff
@@ -0,0 +1,107 @@
+# Fabric Network Operator
+
+- A tool to launch fabric network on kubernetes cluster or local machine with a docker-compose
+file using a network input file and gives back connection profiles for each peer organization
+to use with any client. This tool also uses a testclient to perform the fabric operations which
+includes creating a channel, joining peers to a channel, anchor peer update, installing, upgrading
+and instantiating the chaincode, performing invokes and queries, based on a test input file. This tool is also used for creating channelTx, migrating the network to etcdraft, checking the health of peers and orderers and upgrading the network on providing a specific network input file. This uses `ytt` 
+<https://github.com/k14s/ytt/blob/master/README.md> to generate all necessary configuration files 
+and a go program to launch fabric network
+
+## Prerequisites
+
+- Go 1.11.0 or above <https://golang.org/dl/>
+- yaml.v2 go package (go get gopkg.in/yaml.v2)
+- Fabric binaries in $PATH <https://hyperledger-fabric.readthedocs.io/en/latest/install.html>
+
```

Comment:
This seems incomplete. Provide more information saying down binaries and keep them in the $PATH

Reply:
I think that's not needed. It seems to be understandable for a user to download the binaries in the $PATH

Reply:
can you just insert a word? Say:
Fabric binaries downloaded in $PATH

Reply:
Done